borrowing/urls.py

```diff
@@ -1,8 +1,16 @@
-from django.urls import path, include
-from rest_framework import routers
+from django.urls import path
 
-urlpatterns = [
+from borrowing.views import (
+    BorrowingSerializer,
+    BorrowingListView,
+    BorrowingDetailView,
+)
 
+urlpatterns = [
+    path("borrowings_list/", BorrowingListView.as_view(), name="borrowing-list"),
```

Comment:
better just borrowings with out list?

Reply:
I told him to do this according documentation